manifests/s/Samsung/EasyPrinterManager/2.00.01.24/Samsung.EasyPrinterManager.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Samsung.EasyPrinterManager
+PackageVersion: 2.00.01.24
```

Comment:
```suggestion
PackageVersion: 2.0.1.24
```

Reply:
![image](https://user-images.githubusercontent.com/29874229/126907699-628456bf-9fbe-490f-ba88-231ab73a8cdb.png)
You sure @ItzLevvie?

Reply:
Ah it didn't show up for me in Control Panel (probably required restarting the VM), I have retracted the suggestion.